solutions/devsprint-bruno-faganello-1/FinanceApp/New Group/FinanceApp+Double.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FinanceApp+Double.swift
+//  FinanceApp
+//
+//  Created by Victor Catão on 17/03/22.
+//
+
+import Foundation
+
+extension Double {
+    func toBRL() -> String? {
+        let formatter = NumberFormatter()
+        formatter.locale = Locale(identifier: "pt_BR")
```

Comment:
[Request Changed]
Pass this identifier as parameter and add a default value as pt_BR